STARTERKIT/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "drupal/skilld-zen",
+    "description": "Skilld Zen theme fork.",
+    "type": "drupal-theme",
+    "license": "GPL-2.0",
+    "require": {
+        "oomphinc/composer-installers-extender": "^1.1",
+        "fxp/composer-asset-plugin": "~1.3",
+        "npm-asset/slick-carousel": "^1.7"
```

Comment:
are you sure that carousel is required?

Reply:
Yes, because in all projects we install this librarie and write base configuration  for it. Also will be ok to add FontAwesome or another font with icons.